contracts/modules/capital/Ramm.sol

```diff
@@ -34,6 +34,7 @@ contract Ramm is IRamm, MasterAwareV2 {
   uint public constant PRICE_BUFFER_DENOMINATOR = 10_000;
   uint public constant GRANULARITY = 2;
   uint public constant PERIOD_SIZE = 86_400; // day
+  uint public constant INITIAL_PERIODS = 9;
```

Comment:
Given it's only used during initialization, I'd declare it inline as we don't need a view function for this.